### ISSUE TITLE
"recieve" is not recognized as misspelled word

### ISSUE THREAD

**Issue by akarmazyan** [barrust/pyspellchecker#112]
Hi,

does anyone know why "recieve" is not detected as misspelled word ?

spell = SpellChecker();
word_list=['recieve', 'mydummy']
misspelled=spell.unknown(word_list)
print (misspelled)
{'mydummy'}

thanks ,

**Comment by barrust** [comment]
We can actually see that it is in the dictionary (erroneously) using the following code:

``` python
from spellchecker import SpellChecker
spell = SpellChecker()
print('recieve' in spell)  #True
print(spell['recieve']) #254
```

So, based on this we can see that the data source used to build the dictionaries included this version of `recieve` 254 times. I will add this misspelling to the en_exclude.txt file to ensure that it is not present in future versions of the dictionary. 

If there are others (and there will be), you can add them to an issue until I find a better way to track words to add and remove. You can also remove it from your local dictionary using the `remove` functionality.

**Comment by barrust** [comment]
This exact issue will be resolved in the next release
